packaging/tags.py

```diff
@@ -58,6 +58,10 @@
 
 
 class Tag(object):
+    """
+    A representation of the tag triple for a wheel. Instances are considered immutable
+    and thus are hashable. Equality checking is also supported.
```

Comment:
```suggestion
    A representation of the tag triple for a wheel.
    
    Instances are considered immutable and thus are hashable. Equality checking is also supported.
```
For docstrings it's good to have a single opening line summarize what something is for and then the details follow in subsequent paragraphs.